File: java/filtering-content/GetItemsOfType.java
// DocSection: filtering_get_items_of_type
// Gets items of content type Product
CompletionStage<ContentItemsListingResponse> items = client.getItems(
  DeliveryParameterBuilder.params()
    .filterEquals("system.type", "product")
    .build()
);
// To use the code for Android projects, see http://docs.kontent.ai/android
// EndDocSection

File: ts/filtering-content/GetItemsOfType.ts
// DocSection: filtering_get_items_of_type
deliveryClient.items<ContentItem>()
  // Gets items based on the type Product
  .equalsFilter("system.type", "product")
  // Same as using .type('product')
  .toObservable()
  .subscribe(response => console.log(response));
// EndDocSection